### PR TITLE
Upcoming

### PR DESCRIPTION
**Pull request by shravnages**
Completed upcoming page

### CODE REVIEW

File: resources/assets/js/components/UpcomingColumn.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
```
Comment (jasonlipo) [inline resources/assets/js/components/UpcomingColumn.js]: Can we put this file in a new folder called Upcoming?

File: resources/assets/js/components/UpcomingColumn.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      this.props.events.length > 0 ?  
+      <div className="column">
+      <div className="column-title">{
+        this.props.start.getDate() == this.props.today.getDate() ? "Today" : 
+          (this.props.start.getDate() == this.props.today.getDate() + 1 ? "Tomorrow" : 
+              this.props.start.getMonth() + 1 + "/" + this.props.start.getDate())
+      }
+      </div>
+      <div className="column-content"> {
+        this.props.events.map((e, i) =>
```
Comment (jasonlipo) [inline resources/assets/js/components/UpcomingColumn.js]: Use `(e, _)` if `i` is not used

File: resources/views/upcoming.blade.php [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+      <div class="header-tab-item tab-selected">Upcoming</div>
+      <a href="/calendar" style="text-decoration:none;"><div class="header-tab-item">Calendar</div><a>
+   @endcomponent
+
+        <div class="width-scrollable">
+
+        <div id="upcoming-widget">
+        </div>
+
+          <!--<div class="column">
```
Comment (jasonlipo) [inline resources/views/upcoming.blade.php]: Need to delete all these commented lines of code

File: resources/assets/js/components/UpcomingColumn.js (hunk 1 above)
Comment (shravnages) [reply resources/assets/js/components/UpcomingColumn.js]: No.

Comment (shravnages) [reply resources/assets/js/components/UpcomingColumn.js]: Ok! Thanks for your suggestion. I am working on it right now.

Comment (shravnages) [reply resources/assets/js/components/UpcomingColumn.js]: Done!

File: resources/views/upcoming.blade.php (hunk 3 above)
Comment (shravnages) [reply resources/views/upcoming.blade.php]: Ok. Thanks for your suggestion!

File: resources/assets/js/components/UpcomingColumn.js (hunk 2 above)
Comment (shravnages) [reply resources/assets/js/components/UpcomingColumn.js]: Ok. Thanks for your suggestion!

Just finished! Great idea, Jason.

**Pull request merged**